src/components/Footer/Footer.css

```diff
@@ -0,0 +1,8 @@
+.footer-container {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 2.5rem;  
+  display: flex;
+  justify-content: space-around;
+}
```

Comment:
Let's think about making this super minimal and make it take up minimal space at the bottom of the page.

Reply:
Sounds like a good idea